.evergreen/ocsp/certs.yml

```diff
@@ -0,0 +1,68 @@
+global:
+  # All subject names will have these elements automatically,
+  # unless `explicit_subject: true` is specified.
+  output_path: '../drivers-evergreen-tools/.evergreen/ocsp/'
```

Comment:
I suggest adding some comment here that refers back to the `README.md` file, even if it's just "See README.md if customizing this path" or something to that effect.